### PR TITLE
Small cleanup & text changes to survey

### PR DESCRIPTION
**Pull request by Tybot204**
I also added a `yarn.lock` for anyone using Yarn to manage packages.

### CODE REVIEW

File: helpers/SurveyLauncher.js [hunk 1]
```diff
@@ -7,14 +7,16 @@ module.exports = class SurveyLauncher{
 
 	async launch(target){
 		let introMessage = [
-			`:exclamation: Hello ${target} :exclamation: Welcome to Cathartes.  We are a Destiny 2 clan that wants to make playing the game more fun.  Here are some quick bullet points on what we are about:`,
+			`:exclamation: Hello ${target} :exclamation: Welcome to Cathartes.
+			 We are a Destiny 2 clan that wants to make playing the game more fun.
+			 Here are some quick bullet points on what we are about:`,
```
Comment (Tybot204) [inline helpers/SurveyLauncher.js]: ~I didn't get a chance to test the display of this yet, but do ES6 templates make newlines anytime the template has newlines, or do you have to specify `\n`? I was only breaking this line up to be more readable and don't want it breaking to multiple lines in Discord.~

EDIT: Nevermind, I found in the docs for templates that it _does_ create line breaks. We'll have to use `+` here with multiple strings I guess...

Comment (Buluphont) [reply helpers/SurveyLauncher.js]: You could also choose to use a string array. Discord.js will concatenate string arrays with newlines when sending them.

File: commands/Demote.js [hunk 2]
```diff
@@ -12,7 +12,7 @@ module.exports = class Demote extends Command{
 		});
 	}
 
-	async execute(msg, args){	// eslint-disable-line no-unused-vars
```
Comment (Buluphont) [inline commands/Demote.js:15]: Please leave this in until args is actually used, else eslint will bitch

File: helpers/SurveyLauncher.js [hunk 3]
```diff
@@ -128,23 +130,29 @@ module.exports = class SurveyLauncher{
...
 
 		if(memberPath){
 			toSend.push("");
 			toSend.push("What’s an Applicant?");
-			toSend.push("Well you haven’t earned the clan tags just yet.  You need to participate in a “Clan Rush Week” in order to earn that badge.  But you can hangout in discord, join our clan members in activities, and come to clan events.  More details on the upcoming Rush Week will be sent to you soon.");
+			toSend.push(
+				`Well you haven’t earned the clan tags just yet.
+				 You need to participate in a “Clan Rush Week” in order to earn that badge.
+				 But you can hangout in discord, join our clan members in activities, and come to clan events.
+				 More details on the upcoming Rush Week will be sent to you soon.`
```
Comment (Buluphont) [inline helpers/SurveyLauncher.js]: Introduced newlines, but you figured this out by now.

**Review by Buluphont — APPROVED** [review]
Looks good, address my comments if you give a shit, merge it straight in otherwise because I don't care lol. Those eslint suppress flags are nice if you actually use eslint though, and I'm not sure it's right to just strip out the param because we could very easily one day need access to it. I'unno, I'm not a jsdev

File: commands/Demote.js (hunk 2 above)
Comment (Tybot204) [reply commands/Demote.js:15]: @Buluphont `args` is used here, first on line 23. I ran ESLint through the whole project on commit so we should be good 👍. I kept all the flags like this where `args` wasn't used yet!

Comment (Buluphont) [reply commands/Demote.js:15]: Oh sick couldn't tell from the PR view. Awesome.

**Pull request merged**